primitives/trie/src/storage_proof.rs

```diff
@@ -58,6 +58,11 @@ impl StorageProof {
 		StorageProofNodeIterator::new(self)
 	}
 
+	/// Convert into plain node vector.
+	pub fn into_nodes(self) -> Vec<Vec<u8>> {
```

Comment:
So we need this to be able to encode Vec of node in protobuff frame.
Makes me wonder if we should treat storage proof in the protobuf protocol as a single byte array (so any kind of proof can be attached), no strong opinion about it.

Reply:
Changed